tools/state-viewer/src/cli.rs

```diff
@@ -512,3 +532,23 @@ impl ViewTrieCmd {
         view_trie(store, hash, self.shard_id, self.shard_version, self.max_depth).unwrap();
     }
 }
+
+#[derive(Parser)]
+pub struct ViewTrie2Cmd {
```

Comment:
`ViewTrie2` isn't a user-friendly name :)

Taking a step back. Can this be merged with the `ViewTrie` command? For example, as a subcommand to choose between the two behaviors?

Reply:
I merged both into a single command and added --format to choose between full or pretty formatting.